dockerfiles/testsuite/env.miner2

```diff
@@ -1,6 +1,6 @@
 VOCDONI_DATADIR=/app/run
 VOCDONI_MODE=miner
-VOCDONI_LOGLEVEL=info
+VOCDONI_LOGLEVEL=debug
```

Comment:
i'd leave both miners in info loglevel, 
else this clutters the CI log, when there's a failure